setup.py

```diff
@@ -27,30 +27,48 @@
 import os
 import sys
 
-from setuptools import setup
+from setuptools import find_packages, setup
 from setuptools.command.test import test as TestCommand
 
 readme = open('README.rst').read()
 history = open('CHANGES.rst').read()
 
-requirements = [
-    'Flask>=0.10.1',
-    'invenio-base>=0.3.0',
-    'invenio-ext>=0.2.1',
-    'invenio-upgrader>=0.1.0',
-    'invenio-utils>=0.1.1',
+tests_require = [
+    'check-manifest>=0.25',
+    'coverage>=4.0',
+    'isort>=4.2.2',
+    'pep257>=0.7.0',
+    'pytest-cache>=1.0',
+    'pytest-cov>=1.8.0',
+    'pytest-pep8>=1.0.6',
+    'pytest>=2.8.0',
+    'MySQL-python>=1.2.5',
+    'funcsigs>=0.4',
 ]
 
-test_requirements = [
-    'pytest>=2.8.0',
-    'pytest-cov>=2.1.0',
-    'pytest-pep8>=1.0.6',
-    'coverage>=4.0.0',
+extras_require = {
+    'docs': [
+        "Sphinx>=1.3",
+    ],
+    'tests': tests_require,
+}
+
+extras_require['all'] = []
+for reqs in extras_require.values():
+    extras_require['all'].extend(reqs)
+
+setup_requires = [
+    'Babel>=1.3',
 ]
 
+install_requires = [
+    'Flask-BabelEx>=0.9.2',
```

Comment:
I think there's a couple of dependencies missing here - e.g. ``invenio-db>=1.0.0a2``